README.md

```diff
@@ -189,6 +189,7 @@ Community Maintained Cars and Features
 | Volkswagen| Atlas 2018-19                 | Driver Assistance | Stock            | 0mph               | 0mph         |
 | Volkswagen| e-Golf 2014, 2019-20          | Driver Assistance | Stock            | 0mph               | 0mph         |
 | Volkswagen| Golf 2015-19                  | Driver Assistance | Stock            | 0mph               | 0mph         |
+| Volkswagen| Golf Sportwagon 2015          | Driver Assistance | Stock            | 0mph               | 0mph         |
```

Comment:
Alpha sort below Golf SportsVan, and use the VW official spelling/style "SportWagen" (capital-W, wagen with an E instead of O). Other than that, LGTM!

Reply:
corrected it.
Thanks for your patience. I'm new to openpilot and github.